media/js/firefox/family-nav.js

```diff
@@ -10,7 +10,263 @@ if (typeof window.Mozilla === 'undefined') {
 ;(function(Mozilla, $) {
     'use strict';
 
-    Mozilla.FxFamilyNav = (function() {
+    var $fxFamilyNavWrapper = $('#fxfamilynav-wrapper');
```

Comment:
This file has been reverted to an old state. Please update this file to match what is currently in master/production, then add the GTM stuff.